flipper/__init__.py

```diff
@@ -0,0 +1,55 @@
+import cv2
+
+
+class Flipper(object):
+    """Class to encapsulate all the flipper logic.
+
+    Intended flow is:
+
+        left_flipper = Flipper('left')
+        while(frame):
+            contours = cv2.findContours(...)
+            left_flipper.add_contours(contours)
+            ...
+        left_flipper.train()
```

Comment:
Can train() return true or false? Like if there is enough data, it says "true." If not enough, it would say false.
I'd like to implement this logic: Add contour, train. Not enough data? Add contour, train. Not enough data? ... you got the idea. Then I don't have to track how many I need to add, just if it's trained or not. But you'd need to keep the array to a limited size, so that if there is a botched frame, you discard it, instead of it ruining training experience.